Editor/EditorCore/ProBuilderEditor.cs

```diff
@@ -49,7 +49,6 @@ public sealed class ProBuilderEditor : EditorWindow, IHasCustomMenu
         static ProBuilderToolManager toolManager => s_Instance != null ? s_Instance.m_ToolManager : null;
         internal EditorToolbar toolbar => m_Toolbar; // used by unit tests
         static ProBuilderEditor s_Instance;
-        static Pref<SelectMode> s_LastActiveSelectMode = new Pref<SelectMode>("editor.lastActiveSelectMode", SelectMode.Face);
```

Comment:
Should add `"editor.lastActiveSelectMode"` to `PreferenceUpdater.DeleteObsoletePreferences` to clean up unused pref keys.

Reply:
this change has now been made and pushed